backstop/tests/expandable-block.html

```diff
@@ -23,6 +23,15 @@
       href="../assets/demo.css"
     />
     <script src="../assets/theme.js"></script>
+    <script>
+      customElements.define('expandable-content', class extends HTMLElement {
+        connectedCallback() {
+          const innerHTML = `<div class="iui-expandable-content"><div>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</div></div>`;
```

Comment:
Maybe have just have Lorem ipsum part here and leave `iui-expandable-content` as it was.

Reply:
Done